lib/client/api.go

```diff
@@ -3217,6 +3261,44 @@ func ParseLabelSpec(spec string) (map[string]string, error) {
 	return labels, nil
 }
 
+// ParseSearchKeywords parses a string ie: foo,bar,"quoted value"` into a slice of
+// strings: ["foo", "bar", "quoted value"].
+// Almost a replica to ParseLabelSpec, but with few modifications such as
+// allowing a custom delimiter. Defaults to comma delimiter if not defined.
+func ParseSearchKeywords(spec string, customDelimiter rune) []string {
+	delimiter := customDelimiter
+	if customDelimiter == 0 {
```

Comment:
Same suggestion as above about `customFilter`
```suggestion
	if delimiter == 0 {
```